saleor/checkout/views/summary.py

```diff
@@ -24,9 +25,10 @@ def create_order(checkout):
     checkout.clear_storage()
     checkout.cart.clear()
     user = None if checkout.user.is_anonymous else checkout.user
-    order.create_history_entry(user=user, content=pgettext_lazy(
-        'Order status history entry', 'Order was placed'))
-    order.send_confirmation_email()
+    order.history.create(
+        order=order, user=user, content=pgettext_lazy(
```

Comment:
You don't have to pass `order` as an argument here. Calling `create` on `history` will automatically relate this entry with the order ([docs](https://docs.djangoproject.com/en/2.0/topics/db/queries/#additional-methods-to-handle-related-objects)).

Reply:
Ok, removed here and in other places.